### ISSUE TITLE
Slashes don't escape properly

### ISSUE THREAD

**Issue by Flynsarmy** [shazahm1/WP-Syntax#19]
I have the following code

``` html
<pre lang="bash" line="1">rename "s/ ([a-z])/ \U\1\E/g" *.mkv</pre>
```

and on first page load it will show correctly as `s/ ([a-z])/ \U\1\E/g" *.mkv` however on second page load it shows as `s/ ([a-z])/ U1E/g" *.mkv` - notice the `\` characters are now missing.. It appears this has something to do with the caching - I'm guessing the cache is loaded then the string is escaped again.

**Comment by shazahm1** [comment]
Actually the slashed are being stripped when saved into the postmeta cache.

**Comment by shazahm1** [comment]
Using wp_slash() highlighted post content when updating the postmeta resolves this.

**Comment by shazahm1** [comment]
Fixed in 11a7d96d12ade00c8281b7a6bad98dcc41a94eeb.
